e2e/wdio/core/tests/form-message.e2e-spec.ts

```diff
@@ -0,0 +1,153 @@
+        expect(getText(messageInformation)).toBe(clickEventMessage1);
+    });
+
+    fit('should check message with input group field', () => {
+        scrollIntoView(messageWithInputGroup, 1);
+        setValue(messageWithInputGroup, testText, 1);
+        expect(getValue(messageWithInputGroup, 1)).toBe(testText);
+        click(buttons, 1);
+        expect(getText(messageInformation)).toBe(clickEventMessage2);
+    });
+
+    fit('should check message with input group field - hover', () => {
+        scrollIntoView(messageWithInputGroup, 2);
+        mouseHoverElement(messageWithInputGroup, 2);
+        expect(getText(messageInformation)).toBe(clickEventMessage3);
+        setValue(messageWithInputGroup, testText, 2);
+        expect(getValue(messageWithInputGroup, 2)).toBe(testText);
+        expect(isElementClickable(buttons, 2)).toBe(true);
+    });
+
+    fit('should check message with textarea', () => {
+        scrollIntoView(messageWithTextArea);
+        setValue(messageWithTextArea, testMultilineText);
+        expect(getValue(messageWithTextArea)).toBe(testMultilineText);
+        expect(getText(messageInformation)).toBe(clickEventMessage4);
+    });
+
+    describe('Check visual regression', function() {
+
+        it('should check examples visual regression', () => {
+            formMessagePage.saveExampleBaselineScreenshot();
+            expect(formMessagePage.compareWithBaseline()).toBeLessThan(1);
+        });
+
+        it('verify input fields states', () => {
+            const inputLength = getElementArrayLength(inputFields);
+            for (let i = 0; i < inputLength; i++) {
+                scrollIntoView(inputFields, i);
+                checkElementStates(inputFields, inputField + i + '-', input, i);
+            }
+        });
+
+        it('verify buttons states', () => {
+            const buttonsLength = getElementArrayLength(buttons);
+            for (let i = 1; i < buttonsLength; i++) {
+                scrollIntoView(buttons, i);
+                checkElementStates(buttons, button + i + '-', btn, i);
+            }
+        });
+
+        it('verify message information 1st input field hover state', () => {
+            scrollIntoView(messageWithInput);
+            click(messageWithInput);
+            checkMessageHoverState(messageInformation, message + '-0', msg);
+        });
+
+        it('verify message information 2nd input field hover state', () => {
+            scrollIntoView(buttons, 1);
+            click(buttons, 1);
+            checkMessageHoverState(messageInformation, message + '-1', msg);
+        });
+
+        it('verify message information textarea field hover state', () => {
+            scrollIntoView(messageWithTextArea);
+            click(messageWithTextArea);
+            checkMessageHoverState(messageInformation, message + '-2', msg);
+        });
+    });
+
+    function checkMessageHoverState(selector: string, tag: string, elementName: string, index: number = 0): void {
```

Comment:
what is the diff between checkElementHoverState and checkMessageHoverState ?
only 'hover-state-' tag part ?